### PR TITLE
Add instruction for validating migration data

### PR DESCRIPTION
**Pull request by teknofire**
### :nut_and_bolt: Description: What code changed, and why?

Add documentation for verify migration 

CHEF-208 and CHEF-351

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md [hunk 1]
```diff
@@ -204,6 +228,41 @@ In Automate HA there are equivalent command which had been used in A2HA:
...
+        knife cookbook list -s $org | wc -l
+        echo -n "total objects: "
+        knife list / -R -s $org | wc -l
+        echo "----------------"
+    done
+    ```
+
+    Compare the pre migration to post migration counts `diff pre_migration_infra_counts.log post_migration_infra_counts.log`
+
+1. Connect Chef-Workstation to the new cluster and use knife to communication with Automate HA
```
Comment (vivekshankar1) [inline components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md:258]:
```suggestion
1. Connect Chef-Workstation to the new cluster and use knife to communicate with Automate HA
```

File: components/docs-chef-io/content/automate/ha_inplace_migration.md [hunk 2]
```diff
@@ -184,6 +208,41 @@ Click [here](/automate/ha_backup_restore_object_storage/) to know more about the
...
+        knife cookbook list -s $org | wc -l
+        echo -n "total objects: "
+        knife list / -R -s $org | wc -l
+        echo "----------------"
+    done
+    ```
+
+    Compare the pre migration to post migration counts `diff pre_migration_infra_counts.log post_migration_infra_counts.log`
+
+1. Connect Chef-Workstation to the new cluster and use knife to communication with Automate HA
```
Comment (vivekshankar1) [inline components/docs-chef-io/content/automate/ha_inplace_migration.md:238]:
```suggestion
1. Connect Chef-Workstation to the new cluster and use knife to communicate with Automate HA
```

**Review by jayvikramsharma1 — APPROVED** [review]

**Review by vivekshankar1 — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md [hunk 3]
```diff
@@ -27,9 +27,33 @@ This page explains migrating the existing A2HA data to the newly deployed Chef A
 ## Prerequisites
 
 - Ability to mount the file system, which was mounted to A2HA Cluster for backup purposes, to Automate HA.
-
 - Configure the A2HA to take backup on a mounted network drive (location example: `/mnt/automate_backup`).
 
+## Capture information about the current A2HA instance
+
+In order to verify the migration is completed successfully we'll need to capture some information about the current installation. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md:34]: Replace it with: We'll need to capture information about the current installation to verify the migration is completed successfully. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.

File: components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md [hunk 4]
```diff
@@ -204,6 +228,41 @@ In Automate HA there are equivalent command which had been used in A2HA:
 | gather logs                | `bash automate-cluster-clt gather-logs `                  | `bash chef-automate gather-logs`                              |
 | workspace                  | `bash automate-cluster-clt workspace `                    | `bash chef-automate workspace [OPTIONS] SUBCOMMAND [ARG] ...` |
 
+## Validate successful migration
+
+1. Check the Automate UI of Automate HA. Check whether the data is present in Automate UI for HA.
+1. If you are using the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server, this should match the counts captured at the start of the migration
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md:234]: Replace it with: 1. If you use the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server. This should match the counts captured at the start of the migration

File: components/docs-chef-io/content/automate/ha_inplace_migration.md [hunk 5]
```diff
@@ -28,11 +28,35 @@ This page explains the In-Place migration of A2HA to Automate HA. This migration
...
 
 - A healthy state of the A2HA cluster to take fresh backup.
-
 - A2HA is configured to take backup on a mounted network drive (location example: /mnt/automate_backup).
-
 - Availability of 60% of space.
 
+## Capture information about the current A2HA instance
+
+In order to verify the migration is completed successfully we'll need to capture some information about the current installation. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_inplace_migration.md:36]: Replace it with: We'll need to capture information about the current installation to verify the migration is completed successfully. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.

File: components/docs-chef-io/content/automate/ha_inplace_migration.md [hunk 6]
```diff
@@ -184,6 +208,41 @@ Click [here](/automate/ha_backup_restore_object_storage/) to know more about the
 1. Reset the backup configuration path to Opensearch so that new backups will be stored in Opensearch directory, please click [here](/automate/ha_backup_restore_file_system/#configuration-for-automate-node-from-provision-host) to know more.
 {{< /note >}}
 
+## Validate successful migration
+
+1. Check the Automate UI of Automate HA. Check whether the data is present in Automate UI for HA.
+1. If you are using the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server, this should match the counts captured at the start of the migration
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_inplace_migration.md:214]: Replace it with: 1. If you use the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server. This should match the counts captured at the start of the migration.

**Review by dishanktiwari2501 — APPROVED** [review]
Left some minor comments.

**Pull request merged**